runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/graph/CreateExecutableStageNodeFunction.java

```diff
@@ -235,6 +210,17 @@ public Node apply(MutableNetwork<Node, Edge> input) {
           sdkComponents.registerCoder(elementCoder);
           RunnerApi.Coder coderProto = CoderTranslation.toProto(elementCoder, sdkComponents);
           componentsBuilder.putCoders(coderId, coderProto);
+          // For now, Dataflow runner harness only deal with FixedWindow.
+          if (FullWindowedValueCoder.class.isInstance(javaCoder)) {
```

Comment:
You didn't address my comment of why you're not doing `javaCoder isinstance FullWindowedValueCoder`. Generally classObject.isInstrance(Object) is only needed if classObject is not known at compile time.